addons/l10n_cn/models/account_move.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    fapiao = fields.Char(string='Fapiao Number', size=8, copy=False, tracking=True)
```

Comment:
Sure you want Char for number?  (think so too, just want to be sure)  Is there any validation on this?